### PR TITLE
Try using the site padding control

### PR DESCRIPTION
**Pull request by kjellr**
For testing against. https://github.com/WordPress/gutenberg/pull/36214

This PR: 

- Removes all left/right padding from the site templates.

### CODE REVIEW

File: theme.json [hunk 1]
```diff
@@ -341,7 +341,8 @@
 			}
 		},
 		"spacing": {
-			"blockGap": "1.5rem"
+			"blockGap": "1.5rem",
+			"padding": "0 var(--wp--custom--spacing--small)"
```
Comment (youknowriad) [inline theme.json]: This should be an object with left, right, top, bottom and not a string.

Comment (youknowriad) [reply theme.json]: It should be moved to `settings.layout.padding` as well.

Comment (kjellr) [reply theme.json]: Ah, ok that gets things working right! Thanks.

**Comment by kjellr** [comment]
I'm going to close this one for now, since it's outdated and the corresponding PR isn't going to be a part of 5.9.

**Pull request closed without merging**